rtrlib/rtr/rtr.h

```diff
@@ -170,5 +204,48 @@ void rtr_stop(struct rtr_socket *rtr_socket);
  */
 const char *rtr_state_to_str(enum rtr_socket_state state);
 
+/**
+ * @brief Checks if a given interval is inside or outside of the predefined ranges.
+ * @param[in] interval Interval value received from a data PDU.
+ * @param[in] minimum Lower bounds of the range.
+ * @param[in] maximum Upper bounds of the range.
+ * @return BELOW_INTERVAL_RANGE If the given interval is below the specified range.
+ * @return ABOVE_INTERVAL_RANGE If the given interval is above the specified range.
+ * @return INSIDE_INTERVAL_RANGE If the given interval is inside the specified range.
+ */
+int rtr_check_interval_range(uint32_t interval, int16_t minimum, int32_t maximum);
```

Comment:
I think its a bit strange that min and max have different types, `int16_t` and `int32_t` specifically, which could be unsigned by the way, right?!